renewer/tasks/renewals.py

```diff
@@ -0,0 +1,74 @@
+from huey import crontab
+
+from renewer.models.cdn import CdnRoute
+from renewer.db import SessionHandler
+from renewer.models.domain import DomainRoute
+from renewer.huey import huey
+from renewer.tasks import alb, cdn, iam, letsencrypt, s3
+
+
+@huey.periodic_task(crontab(month="*", day="*", hour="12", minute="0"))
+def renew_all_certs():
+    routes = []
+    with SessionHandler() as session:
+        for Route in (CdnRoute, DomainRoute):
+            routes.extend(Route.find_active_instances(session))
+        for route in routes:
+            if route.needs_renewal:
+                queue_all_tasks(route, session)
+
+
+def queue_all_tasks(route, session):
+    if isinstance(route, DomainRoute):
+        get_renewal_pipeline = get_domain_renewal_pipeline
+    elif isinstance(route, CdnRoute):
+        get_renewal_pipeline = get_domain_renewal_pipeline
```

Comment:
Looks like this should be `get_cdn_renewal_pipeline` instead?

```suggestion
        get_renewal_pipeline = get_cdn_renewal_pipeline
```